mm/alignment.py

```diff
@@ -391,6 +391,108 @@ def hash(cls, kwargs):
         )
 
 
+class PlotAlignmentJob(Job):
+    """
+    Plots an alignment from its log file.
+    This is an isolate job based on the tasks available in :func:`i6_core.mm.alignment.AlignmentJob.plot`
+    and :func:`i6_core.corpus.filter.FilterSegmentsByAlignmentConfidenceJob.plot`.
+    """
+
+    def __init__(
+        self,
+        alignment_log_files: List[tk.Path],
+        clip_low: float = float("-inf"),
+        clip_high: float = float("inf"),
+        clip_percentile_low: float = 0.0,
+        clip_percentile_high: float = 100.0,
+        zoom_x_min: Optional[float] = None,
+        zoom_x_max: Optional[float] = None,
+        zoom_y_min: Optional[float] = None,
+        zoom_y_max: Optional[float] = None,
+        num_bins: int = 50,
+    ):
+        """
+        :param alignment_log_files: Alignment log files from the alignment job.
+        :param clip_low: Number symbolizing the absolute number at which the plot will be clipped to the left.
+            If given along with any other value restricting the minimum, the most restrictive value will prevail.
+            By default clips at the minimum value. Has priority over :param:`clip_percentile_low`.
```

Comment:
`clip_low = max(self.clip_low, np.percentile(self.clip_percentile_low), min_value)`
this looks like neither `clip_low` nor `clip_percentile_low` have priority

Reply:
Done. I think it's now better expressed. Thanks for the heads up!